docs/src/api/compositecontroller.md

```diff
@@ -251,6 +252,21 @@ new ones from scratch.
 
 [Job]: https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/
 
+## Managing Controller
+
+Metacontroller ensures that all child objects created from your sync hook are
+owned by the parent object, which is best practice when building operators.
+There are some occasions, however, where you may not want Metacontroller to set
+the `controller` field of this `ownerReferences` object to `true`. This is
+usually when you are creating children that will become owned by some other
+controller, for e.g. a cluster-api cluster custom resource; such a resource
+needs to be owned and controller by its own controller.
+
+If you set this to `false`, the children will still get an entry in
+`ownerReferences` pointing to the parent, but the `controller` field will be
+set to `false`. It is not possible to set this value for some children and not
```

Comment:
after writing this i think it would be better to set this value per-resource, in the attachments array for decorators and the children array for composites.